main/midnight/src/ui/uicommandwindow.cpp

```diff
@@ -369,7 +369,7 @@ void uicommandwindow::setCommandText(layoutid_t id, const std::string& value)
     }
 }
 
-void uicommandwindow::addCommandText(Node* node)
+void uicommandwindow::addCommandText(cocos2d::Node* node)
```

Comment:
VS compiler complaining about ambiguous access (see: https://docs.microsoft.com/en-us/cpp/error-messages/compiler-errors-1/compiler-error-c2385?f1url=%3FappId%3DDev16IDEF1%26l%3DEN-US%26k%3Dk(C2385)%26rd%3Dtrue&view=msvc-170). Disambiguation by use of cocos2d:: prefix.

Reply:
Could you try adding the Node in to the top of the uicommandwindow class. You'll see there is already a Widget and Layout there. That's the pattern that I "normally" use! :)

Reply:
Like it. Changed, will be in next push. Thanks!